localized/en-us/games/wordwindow/wordwindow_level.tsv

```diff
@@ -187,55 +187,55 @@ en-US	14	1	5	4	"बच्चे अपनी कक्षा को सजान
+en-US	17	2	5	5	"मार्को ने १२ मछलियाँ खरीदीं। उसने मछलियों की संख्या ४ टैंकों में बराबर कर दी। प्रत्येक टैंक में कितनी मछलियाँ हैं?"	×æ·UUUUôü Ùð 12 ×ÀçÜØæ¡ ¹ÚèÎè¢Ð ©âÙð ×ÀçÜØô¢ ·UUUè â¢wØæ 4 Åñ¢·UUUUô¢ ×ð¢ ÕÚæÕÚ ·UUUUÚ ÎèÐ ÂýˆØð·UUUU Åñ¢·UUUU ×ð¢ ç·UUUUÌÙè ×ÀçÜØæ¡ ãñ¢?	3	4	9	12	fish	fish	fish	fish	12 = <3> + <3> + <3> + <3>	1	3	3	3	3					wordwindow-stone-blue.png	wordwindow-stone-yellow.png	wordwindow-stone-red.png	wordwindow-stone-green.png		
 en-US	17	3	1	5	"आप और आपकी बहन समान रूप से चिप्स के १० बैग साझा करते हैं। चिप्स के कितने बैग प्रत्येक को मिलेंगे?"	¥æÂ ¥æñÚ ¥æÂ·UUUè ÕãÙ â×æÙ MUUUUÂ âð ç¿Œâ ·ðUUU 10 Õñ» âæÛææ ·UUUUÚÌð ãñ¢Ð ç¿Œâ ·ðUUU ç·UUUUÌÙð Õñ» ÂýˆØð·UUUU ·UUUUô ç×Üð¢»ð?	2	3	5	10	bags	bags	bags	bags	10 = <5> + <5>	3	5	5	0	0					wordwindow-stone-red.png	wordwindow-stone-blue.png		
 en-US	17	3	2	5	"सारा ने ६ फूल चुने। उसने फूलों की समान मात्रा में २ फूलदानों में डाल दी। प्रत्येक फूलदान में उसने कितने फूल डालें ?"	âæÚæ Ùð 6 ÈêUUUÜ ¿éÙðÐ ©âÙð ÈêUUUÜô¢ ·UUUè â×æÙ ×æ˜ææ ×ð¢ 2 ÈêUUUÜÎæÙô¢ ×ð¢ ÇæÜ ÎèÐ ÂýˆØð·UUUU ÈêUUUÜÎæÙ ×ð¢ ©âÙð ç·UUUUÌÙð ÈêUUUÜ ÇæÜð¢ ?	2	3	5	6	flowers	flowers	flowers	flowers	6 = <3> + <3>	2	3	3	0	0					wordwindow-stone-yellow.png	wordwindow-stone-blue.png		
-en-US	17	3	3	5	"एक किसान के पास १६ अंडे हैं और वो उन्हें समान रूप से २ ब1से में रखता है। प्रत्येक बॉ1स में कितने अंडे हैं?"	°·UUUU ç·UUUUâæÙ ·ðUUU Âæâ 16 ¥¢Çð ãñ¢ ¥æñÚ ßô ©‹ãð¢ â×æÙ MUUUUÂ âð 2 Õvâð ×ð¢ Ú¹Ìæ ãñÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ ç·UUUUÌÙð ¥¢Çð ãñ¢?	2	4	8	16	eggs	eggs	eggs	eggs	16 = <8> + <8>	3	8	8	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
-en-US	17	3	4	5	"शेन ने एक पेड़ से ६ आम निकाले। उन्होंने आमों की एक समान सं2या को ३ ब1से में रखा। प्रत्येक डि4बे में उसने कितने आम रखे?"	àæðÙ Ùð °·UUUU ÂðÇ¸ âð 6 ¥æ× çÙ·UUUæÜðÐ ©‹ãô¢Ùð ¥æ×ô¢ ·UUUè °·UUUU â×æÙ â¢wØæ ·UUUUô 3 Õvâð ×ð¢ Ú¹æÐ ÂýˆØð·UUUU çÇyÕð ×ð¢ ©âÙð ç·UUUUÌÙð ¥æ× Ú¹ð?	2	3	5	6	mangoes	mangoes	mangoes	mangoes	6 = <2> + <2> + <2>	1	2	2	2	0					wordwindow-stone-red.png	wordwindow-stone-green.png	wordwindow-stone-blue.png		
+en-US	17	3	3	5	"एक किसान के पास १६ अंडे हैं और वो उन्हें समान रूप से २ बक्से में रखता है। प्रत्येक बॉक्स में कितने अंडे हैं?"	°·UUUU ç·UUUUâæÙ ·ðUUU Âæâ 16 ¥¢Çð ãñ¢ ¥æñÚ ßô ©‹ãð¢ â×æÙ MUUUUÂ âð 2 Õvâð ×ð¢ Ú¹Ìæ ãñÐ ÂýˆØð·UUUU Õæòvâ ×ð¢ ç·UUUUÌÙð ¥¢Çð ãñ¢?	2	4	8	16	eggs	eggs	eggs	eggs	16 = <8> + <8>	3	8	8	0	0					wordwindow-stone-blue.png	wordwindow-stone-green.png		
+en-US	17	3	4	5	"शेन ने एक पेड़ से ६ आम निकाले। उन्होंने आमों की एक समान संख्या को ३ बक्से में रखा। प्रत्येक बॉक्स में उसने कितने आम रखे?"	àæðÙ Ùð °·UUUU ÂðÇ¸ âð 6 ¥æ× çÙ·UUUæÜðÐ ©‹ãô¢Ùð ¥æ×ô¢ ·UUUè °·UUUU â×æÙ â¢wØæ ·UUUUô 3 Õvâð ×ð¢ Ú¹æÐ ÂýˆØð·UUUU çÇyÕð ×ð¢ ©âÙð ç·UUUUÌÙð ¥æ× Ú¹ð?	2	3	5	6	mangoes	mangoes	mangoes	mangoes	6 = <2> + <2> + <2>	1	2	2	2	0					wordwindow-stone-red.png	wordwindow-stone-green.png	wordwindow-stone-blue.png		
```

Comment:
Update Chanakya text

Reply:
Chanakya text is correct verified with pramukh font